src/components/shown-data.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import { MyStore } from '../types';
+interface shownDataProps {
+  data: any,
+}
+
+
+const ShownData: React.FC<shownDataProps> = (props)  => {
+  const now = 100;
```

Comment:
Is it some stub? What's the meaning of this 'now'? Why 100 is it's value? It may worth adding a comment on that

Reply:
It was the only testing how the progress bar work and I forgot to delete it.